products/green_fast_track/analyses/pluto_zoning_district_groups.sql

```diff
@@ -0,0 +1,28 @@
+WITH pluto AS (
+    SELECT
+        coalesce(zonedist1, 'NULL') AS zonedist1,
+        coalesce(zonedist2, 'NULL') AS zonedist2,
+        coalesce(zonedist3, 'NULL') AS zonedist3,
+        coalesce(zonedist4, 'NULL') AS zonedist4
+    FROM {{ ref('stg__pluto') }}
+),
+
+grouup_by_rollup AS (
+    SELECT
+        zonedist1,
+        zonedist2,
+        zonedist3,
+        zonedist4,
+        count(*) AS zoning_combo_count
+    FROM pluto
+    GROUP BY DISTINCT ROLLUP (zonedist1, zonedist2, zonedist3, zonedist4) -- noqa: PRS
```

Comment:
Two little things
- "grouup"
- what does "distinct" do in this case? I would imagine that's handled by the "group by" but not super familiar with rollup

Reply:
But from looking at docs it seems redundant

Reply:
lol will fix the typo

[looks like](https://www.postgresql.org/message-id/bf3805a8-d7d1-ae61-fece-761b7ff41ecc@postgresfriends.org) the `distinct` is useful when doing multiple rollups in a single query? I just ran this without it and I don't see a difference in the results so I'll drop it. it's probably an artifact of my confusion and caution when learning to use rollup